Update package to Laravel 5

oriceon opened the pull request:
Hello, make a new repository oauth-5-laravel and use this first commit as a master branch.

I tested facebook, google for now

oriceon committed:
Update package to Laravel 5

oriceon committed:
Update composer.json

oriceon committed:
Update README.md

oriceon committed:
Update README.md

iog3 commented:
Tested and works great for me!

fnfilho commented:
@artdarek any plans on fully supporting this?

iog3 commented:
It needs to go in a separate repo or at least a separate branch as the instructions say to use `dev-master`, so everyone Laravel 4 projects would get updated and the break.

I would leave it as a fork and add it to packagist.

Nidhalios commented:
Here's an updated version for Laravel 5 
https://github.com/oriceon/oauth-5-laravel

Commit:
Correction to package registration with service providers + aliases

DataMechanics commented:
This is given in an arg in the example but there is no scope as to where Request is defined, consequently this breaks everything and makes your hard work useless, please could you clarify in an otherwise great project

oriceon committed:
Merge pull request #1 from tuwannu/patch-1

Correction to package registration with service providers + aliases

armababy committed:
Allow to implement custom storage

oriceon committed:
Merge pull request #2 from armababy/master

[BC] Allow to implement custom storage

hoppities committed:
modify readme to fix session class error

oriceon committed:
Merge pull request #5 from hoppities/master

modify readme to fix session class error

Unyxos committed:
Update to work with Laravel 5.4

The Share() function doesn't work anymore in Laravel 5.4, changed the register() function to make the php artisan vendor:publish work.

oriceon committed:
Merge pull request #6 from Unyxos/patch-1

Update to work with Laravel 5.4

madeITBelgium committed:
Fix undefined method OAuth::consumer() in laravel 5.4

Call to undefined method Artdarek\OAuth\Facade\OAuth::consumer()

oriceon committed:
Merge pull request #7 from madeITBelgium/patch-1

Fix undefined method OAuth::consumer() in laravel 5.4

DDiimmkkaass committed:
Update Lusitanian/PHPoAuthLib to latest version.

oriceon committed:
Merge pull request #8 from DDiimmkkaass/master

Update Lusitanian/PHPoAuthLib to latest version.

DDiimmkkaass committed:
Update Lusitanian/PHPoAuthLib to latest version.

oriceon committed:
Merge pull request #9 from DDiimmkkaass/master

Update Lusitanian/PHPoAuthLib to latest version.

iWedmak committed:
Update composer.json

oriceon committed:
Merge pull request #10 from iWedmak/patch-1

Update composer.json

fh-jashmore committed:
Add provider to vendor publish in README

oriceon committed:
Merge pull request #11 from Fh-jashmore/master

Add provider to vendor publish in README

Commit:
For L5.5+

Support for Laravel Package Auto Discovery

oriceon committed:
Merge pull request #12 from wmandai/patch-1

For L5.5+

MarcosPer committed:
Create OAuthLaravelSession.php

New TokenStorage based on laravel native session

oriceon committed:
Merge pull request #13 from MarcosPer/master

Create OAuthLaravelSession.php

fkupper committed:
Fix PSR standards + quick win code cleaning

oriceon committed:
Merge pull request #14 from fkupper/master

Fix PSR standards + quick win code cleaning
